riotfile.py

```diff
@@ -370,6 +370,50 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 ),
             ],
         ),
+        Venv(
+            name="datastreams",
```

Comment:
Are there any integrations tests that can be added for kafka?

Reply:
It looks like right now, there are only snapshots tests for Kafka? https://github.com/DataDog/dd-trace-py/blob/aa3a538e0b39d77a5ca625c5625ecb1c1a32c108/tests/contrib/kafka/test_kafka.py#L67

Reply:
Could we add a test producing and consuming and checking the stats generated reliably to some degree? Or at the very least the bucket values?